geoips/commandline/log_setup.py

```diff
@@ -206,34 +196,21 @@ def setup_logging(logging_level=None, verbose=True):
         Determines which log formatter will be used. If `True`, a longer format will be
         used, providing more information, but also cluttering the screen. If `False`, a
         shorter format will be used.
-
-    Globals
-    -------
-    logging_setup_done: bool
-        - Whether or not this function has already been called
-    log: logging.RootLogger
-        - A root logging object that has the GeoIPS log format applied. This will only
-          be set once in a program's execution.
     """
-    global logging_setup_done, log
-
-    if not logging_setup_done:
-        log = logging.getLogger()
-        # If logging_level was not specified, default to INTERACTIVE here.
-        if not logging_level:
-            logging_level = "INTERACTIVE"
-        log.setLevel(getattr(logging, logging_level))
-        fmt = logging.Formatter(
-            "%(asctime)s %(module)12s.py:%(lineno)-4d %(levelname)7s: %(message)s",
-            "%d_%H%M%S",
-        )
-        if not verbose:
-            fmt = logging.Formatter("%(asctime)s: %(message)s", "%d_%H%M%S")
+    log = logging.getLogger()
+    # If logging_level was not specified, default to INTERACTIVE here.
+    if not logging_level:
+        logging_level = "INTERACTIVE"
+    log.setLevel(getattr(logging, logging_level))
+    fmt = logging.Formatter(
+        "%(asctime)s %(module)12s.py:%(lineno)-4d %(levelname)7s: %(message)s",
+        "%d_%H%M%S",
+    )
+    if not verbose:
+        fmt = logging.Formatter("%(asctime)s: %(message)s", "%d_%H%M%S")
+    if not log.handlers:
```

Comment:
I think this is just 2 ways of doing the same th9ing - 3either global logging_setup_done variable, or checking if log.handlers is defined.

Reply:
@evrose54?

Reply:
While I implemented the global setup you specified, I can't guarantee that these are the same things. If you're comfortable making this change and can ensure that the logs aren't duplicated, I'm fine with this! Need to do more research on this before I'm positive.

Reply:
@AJThorpe made this change at the same time you did to fix the duplicate errors :-) We've been using this version of the update, so I think they're both fine, the merge just grabbed this version.  I'm gonna just go with it.